src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    <p className="Person__age">
+      {!person.age && (
+        `I am ${person.age}`
+      )}
```

Comment:
Please fix it
```suggestion
      {person.age && (
        `I am ${person.age}`
      )}
```

![image](https://github.com/mate-academy/react_person/assets/70009205/5928393d-e928-4877-a11c-88c393ac72da)